.bones/tasks/marky-hxwf.md

```
@@ -0,0 +1,190 @@
+title: Add LSP-first PreToolUse hook for Read enforcement
+status: closed
+type: task
+priority: 1
+owner: sethyanow@users.noreply.github.com
+---
+
+Add a PreToolUse hook that warns when full-file Read is called on code files >100 lines without offset/limit. Also add Project Rule #9 to CLAUDE.md.
+
+## Design
+
+## Goal
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)